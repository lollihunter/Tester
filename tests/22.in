20 20 20
1 1
7 12
8 9
13 17
20 7
9 15
9 14
17 2
7 17
7 1
9 2
1 20
18 3
19 17
10 12
8 13
11 14
13 13
17 18
20 20
