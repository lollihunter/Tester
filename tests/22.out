9
1 10 11 3 15 17 4 19 20 
